tests/bearlib/aspects/RootTest.py

```diff
@@ -1,12 +1,11 @@
-from coalib.bearlib.aspects import Root, aspectclass
-from coalib.bearlib.aspects.base import aspectbase
+from coalib.bearlib.aspects import Root
+from coalib.bearlib.aspects.meta import isaspect
 
 
 class RootTest:
 
     def test_class(self):
-        assert isinstance(Root, aspectclass)
-        assert issubclass(Root, aspectbase)
+        assert isaspect(Root)
```

Comment:
Not fully sufficient because `isaspect`'s argument can also be an aspect instance